src/modules/assessments/components/AssessmentForm.tsx

```diff
@@ -165,54 +169,64 @@ const AssessmentForm = ({
 
   const canEdit = enrollment?.access.canEditEnrollments;
 
+  usePrintTrigger({
+    startReady: isPrintView,
+    hold: !enrollment,
```

Comment:
The enrollment will always be present when the AssessmentForm is rendered, so I don't think the `hold` param is doing anything. Am I missing something here?

It looks like there is some remnant code in this component from a previous iteration where enrollment wasn't guaranteed (like the `if (!enrollment) return <NotFound />;` below) but we should be able to remove those things. The `enrollment` prop is required on the AssessmentForm, and the IndividualAssessment always passes it.

Reply:
Yeah go for it